src/API/src/models/models.resolver.ts

```diff
@@ -0,0 +1,36 @@
+import { Args, Query, Resolver, Field, ObjectType } from '@nestjs/graphql';
+import { ModelsTransformationService } from './modelsTransformation.service';
+
+export const stringTypeResolver = () => String;
+export const numberTypeResolver = () => Number;
```

Comment:
Feels like these should be in a common place somewhere, we have them in a few resolver files

Reply:
I think you're right - I'll look at refactoring it.

Reply:
I've now factored out the ones in model.resolver.ts - we might want to pull out others from the db ingest process over time.